www/scripts/ionic/css/ionic.css

```diff
@@ -8519,7 +8524,7 @@ progress {
   vertical-align: top;
   text-align: center;
   text-overflow: ellipsis;
-  font-size: 16px;
+  font-size: 2em;
   line-height: 42px;
   cursor: pointer; }
   .button:hover {
```

Comment:
Leave a newline in-between classes, makes the code look prettier and easier to understand.
Also, don't close the brackets in the same line as the last statement, close is in a line of its own, e.g. :
- Wrong way -> `.class { css-statement: value; }
- Right way
.css-class {
        css-rule: value;
}